### PR TITLE
Ohs2020 badge initial support

### PR DESCRIPTION
**Pull request by mwelling**
This has not been tested on the target yet because it doesn't exist yet.

### CODE REVIEW

**Comment by mwelling** [comment]
The board schematic is here for a cross reference:
https://github.com/oshwabadge2020/Badge-PCB/blob/master/schematic.pdf/badge.pdf

File: ports/nrf/boards/ohs2020_badge/mpconfigboard.mk [hunk 1]
```diff
@@ -0,0 +1,10 @@
+USB_VID = 0x239A
+USB_PID = 0x8072
```
Comment (theacodes) [inline ports/nrf/boards/ohs2020_badge/mpconfigboard.mk]: Please request a new PID by filing an issue, here's an example: #2519

File: ports/nrf/boards/ohs2020_badge/mpconfigboard.mk [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+USB_PRODUCT = "OHS2020 Badge"
+USB_MANUFACTURER = "Adafruit Industries LLC"
```
Comment (theacodes) [inline ports/nrf/boards/ohs2020_badge/mpconfigboard.mk]: This should be you/your company.

Comment (mwelling) [reply ports/nrf/boards/ohs2020_badge/mpconfigboard.mk]: Well it is a community design so there was no company but maybe I can just use my company.

Comment (theacodes) [reply ports/nrf/boards/ohs2020_badge/mpconfigboard.mk]: Community name is fine!

File: ports/nrf/boards/ohs2020_badge/mpconfigboard.mk (hunk 1 above)
Comment (mwelling) [reply ports/nrf/boards/ohs2020_badge/mpconfigboard.mk]: https://github.com/adafruit/circuitpython/issues/2529

Comment (mwelling) [reply ports/nrf/boards/ohs2020_badge/mpconfigboard.mk]: Hopefully resolved with the latest commit.

File: ports/nrf/boards/ohs2020_badge/mpconfigboard.mk (hunk 2 above)
Comment (mwelling) [reply ports/nrf/boards/ohs2020_badge/mpconfigboard.mk]: OSHWA it is. Resolved with the latest commit.

**Comment by ladyada** [comment]
i do :) @mwelling can you create a PR for
https://github.com/adafruit/circuitpython-org
as well?

**Comment by mwelling** [comment]
Sure I will do the PR there too.

Any special steps for creating the bootloader?

**Comment by theacodes** [comment]
I can review the circuitpython-org PR quickly if you send it over in the next hour or so.

For the bootloader check out: https://github.com/adafruit/Adafruit_nRF52_Bootloader/pull/102

**Comment by ladyada** [comment]
@mwelling uf2 bootloader is very similar to this process here, just submit the PR to
https://github.com/microsoft/uf2-samdx1

**Comment by ladyada** [comment]
@theacodes 💯  ;)

**Comment by ladyada** [comment]
mergin' cause pre-VID change passed, and current board sub-build passed and im going to 🌮

**Pull request merged**